.devcontainer/devcontainer.json

```diff
@@ -6,7 +6,7 @@
 	// Use 'forwardPorts' to make a list of ports inside the container available locally.
 	// "forwardPorts": [],
 	// Use 'postCreateCommand' to run commands after the container is created.
-	"postCreateCommand": "npm install && npm run build"
+	"postCreateCommand": "npm install"
```

Comment:
npm run builds take a very long time while setting up the codespace, hence removing it from here.